blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,77 @@
+
+## OS Security Patch channel - How it fares against Node Image Channel
+When comparing OS security patch channel with the Node Image channel, several advantages become apparent:
+
+- **Faster Patching Cadence**: The OS Security Patch channel delivers updates 1 to 2 weeks faster than the Node Image channel, addressing critical vulnerabilities (CVEs) promptly. This ensures quicker protection against threats and reduces exposure time.
+- **Minimal Disruptions to workload**: OS security patches focus solely on security packages, avoiding the inclusion of binaries and bug fixes that could potentially disrupt workloads. This targeted approach minimizes disruptions, making it ideal for maintaining Workload Service Level Objectives (SLOs), especially if you are sensitive to dependencies. The OS Security Patch channel reimages approximately 60-70% less frequently than the Auto Node Image channel. In other words, if the Node Image channel reimages 10 times within a certain period, the OS Security Patch channel would likely reimage only 3 or 4 times during the same timeframe.
+- **Better way to handle capacity Constraints**: Every reimage typically results in the creation of 'Surge' nodes, which can be problematic if you are operating in a capacity-constrained region or SKU. The OS Security Patch channel reduces surging by reimaging 60-70% less frequently than the Auto Node Image channel. This makes it an ideal solution for workloads with capacity constraints.
+- **Livepatching and Ubuntu snapshots**: Canonical snapshots are now [seamlessly integrated for Ubuntu in AKS](https://techcommunity.microsoft.com/blog/linuxandopensourceblog/increased-security-and-resiliency-of-canonical-workloads-on-azure---now-in-previ/3970623). With the OS Security Patch channel, Ubuntu cluster nodes automatically download essential security patches and updates from the Ubuntu snapshot service, ensuring predictability and consistency in cloud deployments. AKS rigorously tests these security packages and releases them through safe deployment practices under Azure. The OS Security Patch channel also features Livepatching capabilities for both Ubuntu and Azure Linux systems, significantly reducing the need for reimaging. Reimaging is only necessary for critical updates, such as Kernel package updates, making maintenance more efficient and less disruptive.
```

Comment:
Livepatching usually is associated with kernel livepatching, so I'd be cautious of that since we're not enabling that

Reply:
Removed it and wherever we are calling it out i say it Live security patching.